.github/workflows/build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           git-config-email: github-actions[bot]@users.noreply.github.com
 
       - name: Post gh pages link under Checks
-        if: always()
+        if: github.ref != 'refs/heads/main'
```

Comment:
but we need deploy results, if visual tests fail in main branch

Reply:
we do
![image](https://user-images.githubusercontent.com/9084735/155726989-5e1d030e-c624-486e-9d7b-29b98aaabd6f.png)